data/ui/search-bar.ui

```diff
@@ -3,7 +3,6 @@
   <!-- interface-requires gtk+ 3.16 -->
   <template class="GamesSearchBar" parent="GtkSearchBar">
     <property name="visible">True</property>
-    <signal name="notify::search-mode-enabled" handler="on_search_mode_notify"/>
```

Comment:
I see no need for this as the search bar's entry is already cleared on enabling. Also this interfered with SearchBar.handle_event